src/screens/Home.js

```diff
@@ -98,34 +103,52 @@ export default ({
             />
           </View>
           <View style={styles.heading}>
-            <SectionHeading text="Select a category:" />
+            <SectionHeading
+              text={
+                secondLanguages ? 'Misafidiana sokajy:' : 'Select a category:'
+              }
+            />
```

Comment:
Please create a new file with translation object that will be used in whole app (`\translations\index.js`):



```
// string ids
export const SELECT_CAT = 'SELECT_CAT';
export const LEARN = 'LEARN';

export const LANG_DATA = {
  [SELECT_CAT]: {
    [LANGUAGE_NAMES.EN]: 'Select a category:',
    [LANGUAGE_NAMES.MG]: 'Misafidiana sokajy:',
  },
  [LEARN]: {
    [LANGUAGE_NAMES.EN]: 'Learn',
    [LANGUAGE_NAMES.MG]: 'Minatra',
  },
};

```
then use it like this :
create a const at the top if component for each translated string
pass lang prop from redux state that is equal to native lang selected
 ```

 const selectCatText = LANG_DATA[SELECT_CAT][lang];
  const learnText = LANG_DATA[LEARN][lang];

```

Reply:
repeat same approach for all strings inside the app that needs translations